pkg/broker/broker.go

```diff
@@ -14,6 +15,7 @@ type Subscriber interface {
 
 type Broker interface {
 	Send(room string, message messaging.Message)
+	SendControl(room string, payload interface{})
```

Comment:
I'm not a big fan of this interface addition to the broker:
1) SendControl can be now used to send anything because it accepts interface{}, so it can be literally anything that can be marshaled to JSON, not only control message.
2) I think broker should not have this marshaling. Broker's responsibility is to send messages, it shouldn't be concerned with marshaling. Control message seems just a message, so I don't think we need any new method here. Marshalling should be done somewhere else.